README.md

```diff
@@ -16,7 +16,7 @@ chart](https://github.com/galaxyproject/galaxy-helm).
    dependent Ansible roles.
 
     ```
-    ansible-galaxy install -r requirements_roles.yml -p roles --force-with-deps
+    ansible-galaxy install -r requirements_roles.yml -p roles --force
```

Comment:
I'm running Ansible 2.8.1 and `ansible-galaxy install --help` says the following:
```
-f, --force           Force overwriting an existing role
--force-with-deps     Force overwriting an existing role and it's dependencies
```
Forcing deps install I would think is prefered?